### DISCUSSION TITLE
text reference as in bookdown?

### DISCUSSION THREAD

**Discussion by filippogambarota** [quarto-dev/quarto-cli#7020] [Q&A] [answered]
Could it be possible to use a text reference as in bookdown? (see https://frederikaust.com/papaja_man/tips-and-tricks.html#text-references) where I define a piece of text and then I can reference that for example writing captions at the end of the document.

**Comment by jjallaire** [comment, marked as answer]
No, we don't currently support text-references. That said, you should be able to write captions below computational figures (i.e. outside of the scope of YAML) with some updates to cross references in v1.4. For example:

````markdown
::: {#fig-test}

```{r}
#| echo: false
plot(cars)
```

This is is the caption for the figure.
:::

See @fig-test.

````

Note that you will need to download the v1.4 pre-release to get this functionality: https://quarto.org/docs/download/prerelease.html
